docs/modules/user-guide/partials/proc_adding-a-kubernetes-or-openshift-component-to-a-devfile.adoc

```diff
@@ -1,8 +1,8 @@
-[id="proc_adding-kubernetes-component-to-a-devfile_{context}"]
+[id="proc_adding-a-kubernetes-or-openshift-component-to-a-devfile_{context}"]
 = Adding a `kubernetes` or `openshift` component to a devfile
```

Comment:
In the title, I believe we should use the full, appropriate names for these components: Kubernetes or OpenShift.